Detectors/Raw/include/DetectorsRaw/SimpleRawReader.h

```diff
@@ -0,0 +1,51 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file SimpleRawReader.h
+/// \brief Definition of the simple reader for non-DPL tests
+#ifndef ALICEO2_ITSMFT_SIMPLERAWREADER_H_
+#define ALICEO2_ITSMFT_SIMPLERAWREADER_H_
+
+#include <vector>
+#include <gsl/gsl>
+#include "Headers/RAWDataHeader.h"
+#include "DetectorsRaw/RawFileReader.h"
+
+namespace o2
+{
+
+namespace itsmft
```

Comment:
I guess that namespace should be changed ? (or the location of this file changed)